### PR TITLE
Add `Sec-CH-Prefers-Reduced-Motion`

### PR DESCRIPTION
**Pull request by beaufortfrancois**
https://wicg.github.io/user-preference-media-features-headers/#sec-ch-prefers-reduced-motion

FYI: @tomayac

***
<a href="https://pr-preview.s3.amazonaws.com/beaufortfrancois/client-hints-infrastructure/pull/128.html" title="Last updated on Sep 15, 2022, 8:45 AM UTC (a68f30d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/client-hints-infrastructure/128/6bd54dd...beaufortfrancois:a68f30d.html" title="Last updated on Sep 15, 2022, 8:45 AM UTC (a68f30d)">Diff</a>

### CODE REVIEW

**Review by arichiv — APPROVED** [review]
Approved, will wait 24hrs if anyone else has comment before merge

**Comment by beaufortfrancois** [comment]
@arichiv Looks like we're good.

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**
